src/driver-sinowealth.c

```diff
@@ -1090,6 +1090,9 @@ sinowealth_update_profile_from_config(struct ratbag_profile *profile)
 			led->color = sinowealth_raw_to_color(device, config->breathing1_color);
 			sinowealth_set_led_from_rgb_mode(led, config->breathing1_mode);
 			break;
+		case RGB_NOT_SUPPORTED:
+			/* This should be unreachable. */
+			break;
```

Comment:
better:
```c
case RGB_NOT_SUPPORTED:
default:
 ...
```
This way it's clear you didn't just forget about this effect *and* you get the error message if it ever happens

Reply:
Fixed :)